packages/Metric/src/Score.tsx

```diff
@@ -53,28 +79,48 @@ const Score: React.FunctionComponent<ScoreProps> = ({
     if (forceDecimal) {
         displayValue = metricValue.toFixed(1);
     }
+
     const arrow = isNegative ? (
-        <ArrowDown
-            size={arrowSize}
-            className={cx("ids-score__arrow", "ids-score__arrow--negative", {
-                "ids-score__arrow--selected": isSelected
-            })}
-        />
+        isWorkleap ? (
```

Comment:
way too many ternary operations going on here.
I think it can be simplified or split up.
Since you have a size adapter, I'd remove this isWorkleap condition from here and put it back to how is was.
above you can create two new consts.
one for the icons classes.
The other would be the icon's name.